CONTRIBUTING.md

```diff
@@ -36,6 +36,15 @@ composed of the following members:
 You can reach TSC at [eve-tsc@lists.lfedge.org](mailto:eve-tsc@lists.lfedge.org)
 and also [browse the archives](https://lists.lfedge.org/g/eve-tsc).
 
+You can signup for the [eve-tsc mailing list](https://lists.lfedge.org/g/eve-tsc/)
+and [eve mailing list](https://lists.lfedge.org/g/eve)
+
+You can susbscribe to the [eve-tsc Calendar](https://lists.lfedge.org/g/eve-tsc/calendar)
+and [eve Calendar](https://lists.lfedge.org/g/eve/calendar)
+
+You can subscribe to slack channels [eve and eve-help](https://lists.lfedge.org/g/eve)
```

Comment:
My bad: the slack channel link should be https://slack.lfedge.org/

Reply:
np, amended the commit